Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.4
```

Comment:
I'm not sure if we want a Dockerfile. If so, we should probably contribute it to https://github.com/voxpupuli/modulesync_config/tree/master/moduleroot